api/views/misc.py

```diff
@@ -32,9 +34,17 @@ def generate_summary() -> Dict:
 
         :return: A dictionary containing the three key-value pairs as described
         """
+        date_minus_two_weeks = datetime.datetime.now(tz=pytz.utc) - datetime.timedelta(
+            weeks=2
+        )
         return {
             "volunteer_count": BlossomUser.objects.filter(is_volunteer=True).count()
             - 2,
+            "active_volunteer_count": Submission.objects.filter(
```

Comment:
Will this act as a set and filter out folks who have completed multiples in the last two weeks?

Reply:
The `.values("completed_by")` essentially groups the submissions by the volunteers. So yes that shouldn't be a problem. If you want I can also add an extra submission to the test cases to verify this behavior.

Reply:
Let's keep the existing test as it is and add one new test for it. Then we'll merge 👍

Reply:
It indeed didn't work as expected, good thinking!
Should be fixed now though.